src/Resources/config/config.yaml

```diff
@@ -2,3 +2,7 @@ imports:
     - { resource: "sylius/resources.yaml" }
     - { resource: "sylius/grid.yaml" }
     - { resource: "sylius/fixtures.yaml" }
+
+sylius_labs_doctrine_migrations_extra:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** these lines work. However, as suggested in the slack discussion, wouldn't we use the `PrependDoctrineMigrationsTrait` in `MonsieurBiz\SyliusCmsPagePlugin\DependencyInjection\ MonsieurBizSyliusCmsPageExtension `? 

```php
protected function getMigrationsNamespace(): string
    {
        return 'MonsieurBiz\SyliusCmsPagePlugin\Migrations';
    }

    protected function getMigrationsDirectory(): string
    {
        return '@MonsieurBizSyliusCmsPagePlugin/Migrations';
    }

    protected function getNamespacesOfMigrationsExecutedBefore(): array
    {
        return ['Sylius\Bundle\CoreBundle\Migrations'];
    }
```

we could probably delete the preprend method : https://github.com/monsieurbiz/SyliusCmsPagePlugin/blob/v1.0.0-RC.14/src/DependencyInjection/MonsieurBizSyliusCmsPageExtension.php#L45-L53